content/enterprise/system-overview/capacity.mdx

```diff
@@ -72,3 +74,6 @@ This resource is harder to predict than memory or CPU usage because it varies pe
 but in general we suggest 35 IOPS per concurrent Terraform run. So if an instance is configured for
 20 concurrent runs, the disk should have 700 IOPS allocated. For reference, on AWS, an EBS volume
 with an allocated size of 250 GB comes with a steady state of 750 IOPS.
+
+It is recommended to use a disk with a minimum of 500 IOPS or 100+MB/s throughput, whichever is the smaller and these are provisioned values.
```

Comment:
Where does this recommendation come from?

Reply:
https://support.hashicorp.com/hc/en-us/articles/4409044739859-Container-ptfe-base-startup-failed

Reply:
Solution 2

Reply:
I'm an engineer on the Terraform Enterprise team and I haven't seen any official recommendation for IOPS and/or throughput aside from the paragraph right above this.

> This resource is harder to predict than memory or CPU usage because it varies per Terraform module, but in general we suggest 35 IOPS per concurrent Terraform run. So if an instance is configured for 20 concurrent runs, the disk should have 700 IOPS allocated. For reference, on AWS, an EBS volume with an allocated size of 250 GB comes with a steady state of 750 IOPS.

In our load testing, we've generally used a disk type that favors I/O, such as `io2` (AWS) with 700 IOPS. 

However, to stay consistent with Support and since this is a minimum recommendation, I would recommend this wording.

```suggestion
It is recommended to use a disk with at least 500 IOPS or 100+ MB/s throughput, whichever is lower.
```